src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,34 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final Injector injector = Injector.getInstance("mate.academy");
 
+    public static void main(String[] args) {
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+        Book firstBook = new Book("Clear code", BigDecimal.valueOf(200.00));
+        Book secondBook = new Book("Effective java", BigDecimal.valueOf(356.47));
+        Book thirdBook = new Book("Head first", BigDecimal.valueOf(124.45));
+        // initialize field values using setters or constructor
+        Book firstBookDB = bookDao.create(firstBook);
+        Book secondBookDB = bookDao.create(secondBook);
+        Book thirdBookDB = bookDao.create(thirdBook);
+        // test other methods from BookDao
+        Book bookByID = bookDao.findById(thirdBookDB.getId())
```

Comment:
There is a missing semicolon at the end of this line. This will cause a syntax error. Please add a semicolon to the end of the line to fix this issue.

Reply:
i have a semilicon on line 24